app/registries/foreman/plugin.rb

```diff
@@ -37,11 +38,12 @@ class PluginRequirementError < Foreman::Exception; end
   class Plugin
     @registered_plugins = {}
     @tests_to_skip = {}
+    @importer_registry = Plugin::ImporterRegistry.new
     @report_scanner_registry = Plugin::ReportScannerRegistry.new
 
     class << self
       attr_reader   :registered_plugins
-      attr_accessor :tests_to_skip, :report_scanner_registry
+      attr_accessor :tests_to_skip, :importer_registry, :report_scanner_registry
```

Comment:
nitpick: can we rename the attribute to `fact_importers`? The same goes for the instance accessor. I think it will be more readable:

``` ruby
Foreman::Plugin.fact_importers.get(:puppet)

Foreman::Plugin.register :foreman_puppet do
  fact_importers.register(:puppet, PuppetFactImporter, true)
end
```

Then we can remove the `delegate` from here and the `alias` from the registry class. They makes me a bit uncomfortable, and I think they won't be needed if we name the accessors better.